docs/reasons.rst

```diff
@@ -0,0 +1,120 @@
+.. _reasons:
+
+===============
+Why PyScaffold?
+===============
+
+|
+
+Stable and battle-proven
+    PyScaffold was created in 2014 to make the developers' life easier.
+    Since then it have been used to create awesome Python packages for data
+    science, industrial automation, academic research, telecom, web development
+    and many other sectors.
+
+Constantly evolving
+    Stability should not come with the price of stagnation.
+    Throughout its existence, PyScaffold have adapted itself to better solve
+    the needs of its users and evolved to accommodate the best practices and
+    standards of the Python ecosystem.
+    In every single major release we offered a clear update path for our users
+    and tried to automate things as much as possible, so everyone can benefit
+    from PyScaffold's improvements without being afraid of breaking things.
+
+Thoroughly tested
+    PyScaffold have an extensive automated test suite that runs for all major
+    operating systems and versions of Python in every commit or pull request.
+    Moreover, PyScaffold is used by its maintainers in their day-to-day
+    programming and for all PyScaffold's extensions, being constantly
+    scrutinized by human beings.
+
+Do one thing and do it well
+    It might sound old-fashioned, but we like this piece of good old UNIX wisdom.
+    *PyScaffold generates files…* More specifically, PyScaffold generates a very
```

Comment:
More -> more
"... that want to be re-usable and distributed."? How about "PyScaffold creates a perfect project structure with tons of useful defaults and best practices... that's it!". Then your sentence with "It does not attempt to do dependency management because there are dedicated tools exactly for that purpose. And so on.